cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/Tokenizer.java

```diff
@@ -0,0 +1,76 @@
+package com.cloudant.sync.query;
+
+/**
+ * Specify the SQLite FTS
+ * <a target="_blank" href="https://www.sqlite.org/fts3.html#creating_and_destroying_fts_tables">
+ * tokenizer</a> to use for indexing.
+ */
+
+public class Tokenizer {
+
+    /**
+     * The default tokenizer which is equivalent to
+     * {@code new Tokenizer("simple")}
+     */
+    public static final Tokenizer DEFAULT = new Tokenizer("simple");
+
+    // string representation of the tokenizer name eg "porter"
+    public final String tokenizerName;
+
+    // tokenizer arguments eg "en_AU" (valid when tokenizerName="icu")
```

Comment:
I know this isn't published javadoc, but I think this is misleading. Arguments can apply to tokenizers other than `icu`, I think your meaning is that `en_AU` is valid for `icu`.
Maybe
> tokenizer arguments (for example with tokenizerName="icu" a valid argument could be "en_AU")